TCMSApp/src/client/app/layout/add-project.html

```diff
@@ -0,0 +1,22 @@
+<div class="container-fluid add-project-container">
+    <div class="row-fluid">
+        <h1> Create new project </h1>
+
+            <div class="container-fluid" style="margin-top: 10px;">
+                <h4> Project name </h4>
+                <input type="text" name="pr-name" placeholder="e. g. My new project" class="col-md-12" value="" required ng-init="vmChild.projectName=''" ng-model="vmChild.projectName">
+            </div>
+
+            <div class="container-fluid" style="margin-top: 10px;" value="">
+                <h4> Project description </h4>
+                <textarea name="pr-descr" id="pr-descr" rows="3" class="col-md-12" style="resize: none;" required ng-init="vmChild.projectDescription=''" ng-model="vmChild.projectDescription"></textarea>
+            </div>
+
+            <div class="container-fluid" style="margin-top: 10px; margin-bottom: 15px;">
```

Comment:
no one likes inline styles
especially if you're doing margin-top:10px at 3 places.
it can be done better)